src/libraries/System.Runtime/tests/System/Attributes.cs

```diff
@@ -283,27 +283,6 @@ private static void GenericAttributesTestHelper<TGenericParameter>(Func<Type, At
             Assert.Equal(1, closedGenericAttributes.Length);
             Assert.Equal(typeof(GenericAttribute<TGenericParameter>[]), closedGenericAttributes.GetType());
         }
-
-        [Fact]
```

Comment:
Is this just a test, or are you actually removing tests because they fail on Mono?

Reply:
Based on the discussion [here](https://github.com/dotnet/runtime/pull/65237#discussion_r808277460) it seemed like these weren't worth it given the failures on Mono. They're all tests I added in earlier commits, not existing tests.